.changeset/fresh-bugs-squashed.md

```
@@ -0,0 +1,5 @@
+---
+"task-master-ai": patch
+---
+
+Fix issue where `expand --all` command doesn't load complexity report in non-default tag contexts. Address bug where `show` command ignores `--tag` flag, defaulting to the current context. 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)